docs/src/source-types.md

```diff
@@ -20,6 +20,9 @@ Cassandra:
   # * when true: the connection is dropped.
   # * when false: the connection will wait until a connection can be made within the limit.
   hard_connection_limit: false
+ 
+  #  Timeout in seconds after which to terminate an idle connection. This field is optional, if not provided, idle connections will never be terminated.
```

Comment:
Should we use a single space after the `#`?

Reply:
Yep, that wasn't intentional.